Xamarin.Forms.Platform.UAP/ButtonRenderer.cs

```diff
@@ -55,7 +55,8 @@ protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
 				if (Element.IsSet(Button.CornerRadiusProperty) && Element.CornerRadius != (int)Button.CornerRadiusProperty.DefaultValue)
 					UpdateBorderRadius();
 
-				if (Element.IsSet(Button.PaddingProperty) && Element.Padding != (Thickness)Button.PaddingProperty.DefaultValue)
+				// By default Button loads width padding 8, 4, 8 ,4
+				if (Element.IsSet(Button.PaddingProperty))
```

Comment:
If you try to clear the Padding property later, it will not clear the padding because of this check.

Reply:
nm